tests/pages/checkout_page.py

```
@@ -157,4 +158,142 @@ def complete_checkout(self, item_name):
+    def set_quantity(self, target, item_name):
+
+        quantity_locator = (
+            By.XPATH,
+            f"//div[contains(.,'{item_name}')]/ancestor::div[contains(@class,'MuiCard')]//p[@data-testid='test-id-quantity']"
+        )
+
+        def get_qty():
+            try:
+                el = self.driver.find_element(*quantity_locator)
+                return int(el.text.strip())
+            except:
```

Ruff (0.15.9)

[error] Do not use bare `except`

(E722)